fdbserver/masterserver.actor.cpp

```diff
@@ -494,11 +494,9 @@ ACTOR Future<Void> newSeedServers(Reference<MasterData> self,
 			tag.id++;
 			idx++;
 
+			// Note that we don't add the private team here. Private team creation is done when system metadata is
+			// created in seedShardServers().
 			servers->emplace_back(newServer.get().interf, seedTeamId);
-
-			if (SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {
-				servers->emplace_back(newServer.get().interf, newServer.get().interf.id());
```

Comment:
not sure why we had duplicate, but do we want to keep the `if`?

Reply:
The above `servers->emplace_back(newServer.get().interf, seedTeamId);` also used in non-ptxn code path. So not sure where to add the if